lib/portage/checksum.py

```diff
@@ -442,18 +442,22 @@ def _apply_hash_filter(digests, hash_filter):
     """
 
     verifiable_hash_types = set(digests).intersection(hashfunc_keys)
+    verifiable_hash_types.discard(size)
```

Comment:
I'm looking at this and trying to figure out how could it work, given that `size` is not defined here.

Reply:
My mistake (forgot quotes).